### ISSUE TITLE
Behaviour of acceptToken

### ISSUE THREAD

**Issue by renjithsasidharan** [florianheinemann/passwordless#123]
acceptToken does not throw any error if the token is invalid, so how do you validate a token that is received via a delivery method (ex: SMS)?

**Comment by friko16** [comment]
true, have the same issue. What I am doing is I am checking if req.user is set, which is done on success. But it shouldn't be handled that way, Error should be thrown.
